src/apps/myaccount/app.js

```diff
@@ -786,6 +786,45 @@ define(function(require) {
 			}
 		},
 
+		validateAccountAdministratorForm(formAccountAdministrator, callback) {
```

Comment:
Please don't use object method shorthand as our build does not include a transpiling step.
```suggestion
		validateAccountAdministratorForm: function(formAccountAdministrator, callback) {
```

Reply:
Just pushed up a fix for this